src/frontend/interfaces/action/build-action-click-handler.ts

```diff
@@ -65,7 +67,17 @@ export const buildActionClickHandler = (
 
     // Default behaviour - you're navigated to action URL and logic is performed on it's route
     if (href) {
-      navigate(href)
+      const url = new URL(`relative:${href}`)
```

Comment:
appending pathname with `<string>:` is a hack which allows us to use `new URL(...)` if we only have a relative url

Reply:
👍🏻 Nice, I didn't know that.